core/tests/puppeteer-acceptance-tests/blog-admin-tests/create-and-delete-draft-blog-post.js

```diff
@@ -0,0 +1,38 @@
+// Copyright 2023 The Oppia Authors. All Rights Reserved.
```

Comment:
What is the collection.svg file above this for?

If it is for creating a post then why are you naming it collection.svg? That gives the impression that it's linked to a collection on the site, which is misleading.

Reply:
yes.. this can be `blog-post-thumbnail.svg`

Reply:
Btw, should we place do file in some folder (maybe images), it seems a bit weird that it is with other files.

Reply:
yes, added an images folder for it.